.github/workflows/build-windows.yml

```diff
@@ -213,6 +213,14 @@ jobs:
           # CMake exits with non-0 status if there are any warnings during the build, so
           cmake --build $env:BUILD_BUILDDIRECTORY\tiledb  -j $env:NUMBER_OF_PROCESSORS --target tiledb_unit --config Release -- /verbosity:minimal
 
+          if( ($env:MAGIC + "X") -eq "X") {
+            if(Test-Path "$env:BUILD_BUILDDIRECTORY/externals/install/bin/magic.mgc") {

```

Comment:
We need this functionality to work in release binaries on all platforms. Hard-coding the path as an environment variable in CI does not look like the correct approach to make that possible. Am I missing something here?